components/script/dom/bindings/interface.rs

```diff
@@ -128,8 +129,8 @@ impl InterfaceConstructorBehavior {
 pub type TraceHook = unsafe extern "C" fn(trc: *mut JSTracer, obj: *mut JSObject);
 
 /// Create a global object with the given class.
-pub unsafe fn create_global_object(
-    cx: *mut JSContext,
+pub fn create_global_object(
```

Comment:
We still take a `*const libc::c_void` argument, so this method should still be marked unsafe, unfortunately.